customer-commons/src/test/java/no/unit/nva/customer/model/CustomerListTest.java

```diff
@@ -33,4 +33,10 @@ void customerListCanBeConvertedToJsonAndBack() {
         CustomerList mappedCustomerList = CustomerList.fromString(customerListJson);
         assertNotNull(mappedCustomerList);
     }
+
+    @Test
+    void stupidJacocoTestForCoverage() {
+        var cust = new CustomerList().getCustomers();
+        assertEquals(Collections.emptyList(),cust);
```

Comment:
Is there really no better way of reaching this code? If not, this looks much better for testing empty collections:

`        assertThat(list, IsEmptyIterable.emptyIterable());
`